standards/source-code.md

```diff
@@ -71,6 +71,9 @@ Surrey, TW9 4DU.
 ### Readme
 A `README.md` should be added to initialise the repository. As work begins on the repository, the README must adhere to [FFC documentation standards](documentation-standards.md).
 
+### Commit Signing
+Signed commits will need to be enforced in the main branch.
```

Comment:
This should be added as another bullet in the `Branch policies` section above.

Can just say, must require signed commit